dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf

```diff
@@ -64,6 +64,10 @@ zone={{account.zone}}, \
 {%            elif account.service == 'dns-o-matic' %}
 protocol=dyndns2, \
 server=updates.dnsomatic.com, \
+{%            elif account.service == 'freedns' %}
+use=web, web=https://freedns.afraid.org/dynamic/check.php, \
```

Comment:
probably best omit the location where to fetch the address from as it's already in the list

https://github.com/opnsense/plugins/blob/b1636198a6629bb433d0a97806bc46936fbe0044/dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf#L22

Reply:
Yes, you are right. Copy paste from my development system and forgot to remove it.